README.md

```diff
@@ -1,12 +1,14 @@
 Deploy a replicaset to coreos like a boss.
-
 Auto-discover new members via etcd.
 
 ## Deploy
 
 ```
+fleetctl destroy mongo@{1..3}.service  mongo-replica-config.service 
+
 etcdctl set /mongo/replica/name myreplica
-fleetctl start mongo@{1..3}.service mongo-replica-config.service
+
+fleetctl start mongo-data@{1..3}.service mongo@{1..3}.service mongo-replica-config.service
```

Comment:
Oh. that was a typo. I forked this repo to other one to use volume container(https://github.com/19hz/coreos-mongodb-cluster). 

I deleted mongo-data@.service related lines from README.